src/lib/plot/axis/axis.js

```diff
@@ -71,6 +72,10 @@ const defaultProps = {
   orientation: BOTTOM
 };
 
+const predefinedClassName = 'rv-xy-plot__axis';
+const VERTICAL_CLASS_NAME = 'rv-xy-plot__axis-vertical';
```

Comment:
`--vertical` is the correct BEM convention. Do you think we'll get requests for separate classNames for left vs right?